build/templates/tox-system_tests.ini.mako

```diff
@@ -80,7 +80,7 @@ deps =
 
 depends =
     ${module_name}-coverage: py{35,36,37,38}-${module_name}-system_tests
-% if config['supports_nitclk']:
+% if uses_other_wheel:
```

Comment:
Seems unrelated to PR.

Reply:
It is a fix I came across while working on this. Do you need a separate PR for this one change?

Reply:
Best practice is to make PRs small and single-purpose. It helps keep source control history clean and (we don't take that much advantage of it because this is a small project) makes cherry-picking changes viable.

Reply:
This change is now in master so this is no longer different.